packages/components/src/Button/Button.tsx

```diff
@@ -155,64 +154,11 @@ export function Button(props: ButtonProps) {
   return <Tag {...tagProps}>{buttonInternals}</Tag>;
 }
 
-function ButtonInternals({
-  label,
-  icon,
-  variation = "work",
-  type = "primary",
-  disabled,
-  size = "base",
-}: ButtonProps) {
+function ButtonInternals({ label, icon, size = "base" }: ButtonProps) {
   return (
     <>
-      {icon && (
-        <Icon
-          name={icon}
-          size={size}
-          color={getColor(variation, type, disabled)}
-        />
-      )}
-      <Typography
-        element="span"
-        textCase="uppercase"
-        fontWeight="extraBold"
-        size={getTypeSizes(size)}
-        textColor={getColor(variation, type, disabled)}
-      >
-        {label}
-      </Typography>
+      {icon && <Icon name={icon} size={size} />}
+      <span className={styles.buttonLabel}>{label}</span>
     </>
   );
 }
-
```

Comment:
I'm open to thoughts on whether removing all of this is better or not from a technical perspective; what I want to get us away from is using that `brightness(80%)` filter on hover and focus states, and I think these functions only passing in one color are why we used that filter.

Reply:
Also my thinking around using `buttonLabel` and CSS instead of `Typography`